indico/modules/events/surveys/templates/manage_questionnaire.html

```diff
@@ -1,35 +1,90 @@
+                        {% endtrans %}
+                </a>
             {% endfor %}
         </div>
     </div>
-    <div class="i-box titled no-padding">
-        <div class="i-box-header">
+
+    <div id="questions" class="i-box titled">
+         <div class="i-box-header">
             <div class="i-box-title">
-            {% trans %}List of questions{% endtrans %}
+                {% trans %}Questionnaire{% endtrans %}
             </div>
         </div>
-        <div class="i-box-content">
-            <ul class="group-list with-buttons no-content-before">
-            {% for question in survey.questions %}
-                <li>
-                    <a href="#" data-href="{{ url_for('.edit_question', question) }}"
-                                data-title="{% trans %}Edit question{% endtrans %}"
-                                class="js-question-dialog">
-                        <strong>{{ question.title }}</strong>
-                    </a>
+        <div class="sortblock i-box-content" style="width: 100%">
+            <ul class="group-list no-content-before">
+                {% for question in survey.questions %}
+                <li style="cursor: move">
+                    <table class="dragHandle" style="table-layout: fixed;width: 100%">
+                        <tbody>
+                            <tr>
+                                <td><strong>{{ question.title }}</strong></td>
+                                <td>{{ question.description }}</td>
+                                <td>{{ form_row(preview_form[question.title], orientation='horizontal',
+                                                widget_attrs=preview_widget_attrs) }}</td>
+                                <td>
+                                    <a data-href="{{ url_for('.remove_question', question) }}" class="i-button icon-remove js-remove-question right"></a>
+                                    <a href="#" data-href="{{ url_for('.edit_question', question) }}"
+                                                data-title="{% trans %}Edit question{% endtrans %}"
+                                                data-question-id="{{ question.id }}"
+                                                class="i-button js-question-dialog icon-edit right"></a>
+                                </td>
+                            </tr>
+                        </tbody>
+                    </table>
                 </li>
-            {% else %}
-                <li>{% trans %}No questions defined{% endtrans %}</li>
-            {% endfor %}
+                {% endfor %}
             </ul>
         </div>
     </div>
+    <script>
+        $('.js-remove-question').on('click', function(e) {
+            e.preventDefault();
+            var $this = $(this);
+            var msg = $T('Do you really want to remove this question room from the survey?');
+            new ConfirmPopup($T('Delete this question ?'), msg, function(confirmed) {
```

Comment:
Besides that, can't you use `data-confirm` and `data-title` on the `js-remove-question` element? You shouldn't really need custom JS for this. global.js contains code to create an implicit form with confirmation for a link if you use `data-href`, `data-method`, `data-confirm` and `data-title`.